### PR TITLE
feat(localai): send prompt action

### PR DESCRIPTION
**Pull request by hkboujrida**
Added a piece for LocalAI (https://localai.io/), you can now use prompt and chat completion with localai with various OSS LLM Models like GTP4ALL or Huggingface LLM models, Llama 2 ...

ap-localai

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/localai/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          required: false,
+      })
+  },
+  required: true,
+})
+export const openai = createPiece({
+  displayName: 'LocalAI',
+  description: 'Use LocalAi to generate text',
+  minimumSupportedRelease: '0.5.0',
+  logoUrl: 'https://avatars.githubusercontent.com/u/128192713?s=48&v=4',
```
Comment (abuaboud) [inline packages/pieces/localai/src/index.ts]: Can you please edit to `https://cdn.activepieces.com/pieces/localai.jpeg`

The pr doesn't allow maintainers to edit

File: packages/pieces/localai/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { PieceAuth,Property, createPiece } from '@activepieces/pieces-framework';
+import { askLocalAI } from './lib/actions/send-prompt';
+
+export const localaiAuth = PieceAuth.CustomAuth({
+  
+  description: `
```
Comment (abuaboud) [inline packages/pieces/localai/src/index.ts]: Let's remove the description for the auth

**Review by abuaboud — CHANGES_REQUESTED** [review]

**Comment by abuaboud** [comment]
@hkboujrida Can you please fix the checks or give me an edit access to this PR?

File: packages/pieces/localai/src/index.ts (hunk 2 above)
Comment (hkboujrida) [reply packages/pieces/localai/src/index.ts]: description removed

File: packages/pieces/localai/src/index.ts (hunk 1 above)
Comment (hkboujrida) [reply packages/pieces/localai/src/index.ts]: done

**Pull request merged**